Directory.Build.props

```diff
@@ -404,6 +404,13 @@
   <!-- Import it at the end of the props file to override the OutputPath for reference assemblies and use common directory props -->
   <Import Project="$(MSBuildThisFileDirectory)eng/ReferenceAssemblies.props" />
 
+  <PropertyGroup Condition="'$(IsTestProject)' != 'true' AND '$(IsReferenceAssembly)' !='true'">
+    <!-- Set the documentation output file globally. -->
+    <DocumentationFile Condition="'$(DocumentationFile)' == ''">$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
+    <!-- We ignore all documentation related warnings. -->
+    <NoWarn>1591</NoWarn>
```

Comment:
NoWarn is being fully overriden. It needs to be appended to any previous value 😄 https://github.com/dotnet/corefx/blob/master/eng/ReferenceAssemblies.props#L15

Add $(NoWarn); before 1591.

Reply:
I probably should also add "1573;" before 1591 too and see if that fixes it.